src/routes/(console)/project-[region]-[project]/sites/create-site/deploy/+page.svelte

```diff
@@ -0,0 +1,365 @@
+<script lang="ts">
+    import { goto } from '$app/navigation';
+    import { base } from '$app/paths';
+    import { page } from '$app/state';
+    import { page as pageStore } from '$app/stores';
```

Comment:
we should only use one and the one from the `/state`. `/stores` one is deprecated.